ports/superdungeonmuncher/README.md

```diff
@@ -0,0 +1,25 @@
+# Super Dungeon Muncher
+This is a port of Super Dungeon Muncher by nexy via PortMaster. It uses [gmloader-next](https://github.com/JohnnyonFlame/gmloader-next) by JohnnyOnFlame.
+
+# Installation
+Purchase the game from [Itch.io](https://n-nexy.itch.io/super-dungeon-muncher) or [Steam](https://store.steampowered.com/app/3095800/SUPER_Dungeon_Muncher) and put `data.win` into your `ports/superdungeonmuncher` folder.
+
+## Default Gameplay Controls
+| Button | Action |
+|--|--|
+|D-PAD|Move|
+|LEFT ANALOG|Move|
+|B|Confirm|
+|A|Confirm|
+|X|Confirm|
+|Y|Confirm|
+
+# License
+Gmloader-next is released under the GPLv2 License, see the license file for more information.
+
+# Thanks
+Special thanks to:
```

Comment:
Markdown needs two spaces to denote a newline, otherwise this continues on one line.

Reply:
thank you! changed :)